app/Http/Transformers/ContestTransformer.php

```diff
@@ -17,7 +17,11 @@ public function transform(Contest $contest)
     {
         return [
             'id' => $contest->id,
-            // @TODO: More to come...
+            'campaign_id' => $contest->campaign_id,
+            'campaign_run_id' => $contest->campaign_run_id,
+            'signups_open' => $contest->waitingRoom->isOpen(),
```

Comment:
Not necessary for merge, but this could be placed on a `WaitingRoomTransformer` and [included as a nested resource](http://fractal.thephpleague.com/transformers/#including-data) so you could re-use the waiting room transformer elsewhere as well.